src/main/kotlin/com/hj/leetcode/kotlin/problem2699/Solution.kt

```diff
@@ -0,0 +1,111 @@
+package com.hj.leetcode.kotlin.problem2699
+
+import java.util.*
+
+/**
+ * LeetCode page: [2699. Modify Graph Edge Weights](https://leetcode.com/problems/modify-graph-edge-weights/);
+ *
+ * TODO([Dijkstra's with TC O((E + V) log V) Beats 100% in all languages](https://leetcode.com/problems/modify-graph-edge-weights/discuss/5708699/))
+ */
+class Solution {
+    /* Complexity:
+     * Time O((E+nLog(n))LogE) and Space O(n+E) where E is the size of edges;
```

Comment:
The time complexity should be O((E+n)Log(n)LogE): The for-loop in minPathWeight(..) could be executed O(E) times and each time it may update key, which takes O(Log(n)).

Reply:
Addressed by https://github.com/hj-core/hj-leetcode-kotlin/pull/444/commits/e16a666ae754a7a0e1d356a5f6b3ccc348ae4fe7